cmd/neofs-ir/internal/validate/config.go

```diff
@@ -0,0 +1,213 @@
+package validate
+
+import "time"
+
+type validConfig struct {
+	Logger struct {
+		Level string `mapstructure:"level"`
```

Comment:
Can we just name it `level` and not have any `mapstructure` tags?

Reply:
We can't because the fields of the structure must be exported for `Unmarshal`.